webserver/tests/webserver/src/test/java/io/helidon/webserver/tests/MaxPayloadSizeTest.java

```diff
@@ -100,7 +100,7 @@ void testActualLengthExceededWithPayload() {
                         it.write(PAYLOAD_BYTES);
                         it.write(PAYLOAD_BYTES);
                         it.write(PAYLOAD_BYTES);
-                    } catch (IOException e) {
+                    } catch (IOException | UncheckedIOException e) {
```

Comment:
(Maybe log at `DEBUG` or `TRACE` level with an appropriate guard?)

Reply:
We don't need that for a test like this

Reply:
Derp forgot this was a test